theme/static/style/style.css

```diff
@@ -325,43 +323,93 @@ pre code {
 
 /* Footer Section */
 
-.footer{
+.footer {
     padding: 2% 0;
     display: inline-block;
     font-size: 1.2rem;
 }
 
-.footer-logo{
+.footer-logo {
     float: right;
     margin-right: 1rem;
     vertical-align: middle;
     overflow-wrap: break-word;
 }
 
-.footer-copyright{
+.footer-copyright {
     line-height: 1.8em;
     vertical-align: middle;
     overflow-wrap: break-word;
 }
 
-.footer-copyright a{
+.footer-copyright a {
     text-decoration: underline;
 }
 
-.footer-sociallinks{
+.footer-sociallinks {
     list-style-type: none;
     text-align: right;
 }
 
-.footer-sociallinks li{
+.footer-sociallinks li {
     display: inline-block;
     margin-right: 1.5rem;
 }
 
-.footer-sociallinks a{
+.footer-sociallinks a {
     font-size: 3.5rem;
 }
 
+/* Features Page */
+
+.horizontally-stacked-list {
+    display: flex;
+    float: left;
+}
+
+.w50 {
+    float: left;
+}
+
+.w25 {
+    float: left;
+}
+
+.testplan-img {
+    margin-bottom: 5rem;
+}
+
+.mark-and-status-bugs {
+    float: left;
+    margin-bottom: 5rem;
+}
+
+.call-to-action-field {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+}
+
+.call-to-action-buttons {
+    text-align: center;
```

Comment:
Can this text-align go into the .call-to-action-field definition? If yes then you can remove this additional class.